src/inventory/weapons/ZombieHand.h

```diff
@@ -0,0 +1,44 @@
+/**
+    Created by Mark Tattrie 3/28/2017
+*/
+#ifndef ZOMBIEHAND_H
+#define ZOMBIEHAND_H
+
+#include "MeleeWeapon.h"
+#include "../../audio/AudioManager.h"
+#include <string>
+#include "../../log/log.h"
+#include "../../audio/AudioManager.h"
+#include "../../sprites/Renderer.h"
+#include "../../sprites/SpriteTypes.h"
+
+
+namespace ZombieHandVars {
+    const std::string TYPE          = "ZombieHand";
+
+    const std::string FIRESOUND     = "";
+    const std::string HITSOUND      = "";
+    const std::string RELOADSOUND   = EFX_WRELOAD01;
+    const std::string EMPTYSOUND    = EFX_PDROP01;
+
+    constexpr int RANGE             = 100;
+    constexpr int DAMAGE            = 20;
+    constexpr int AOE               = 1;
+    constexpr int PENETRATION       = 1;
+
+    constexpr int CLIP              = 10000;
+    constexpr int CLIPMAX           = 10000;
+    constexpr int AMMO              = 10000;
+
+    constexpr int RELOADDELAY       = 0;
+    constexpr int FIREDELAY         = 600;
+}
```

Comment:
these are variables that shouldnt change but please avoid space aligning like this as per the guide